.evergreen.yml

```diff
@@ -49,17 +49,6 @@ functions:
           pip freeze --local --requirement ${requirements_file} > pip.freeze
           diff ${requirements_file}.freeze pip.freeze
 
-  upload code coverage:
```

Comment:
For some reason, the code coverage is broken. I'm just going to remove it since we are really using it.